data/youtube.js

```diff
@@ -10,53 +10,117 @@ const PLAYLIST_NAME_AVOID_LIST = [
   'Family Ministry',
   'Latest for Echo.Church',
 ];
+
+const PLAYLIST_PARTS = ['id', 'contentDetails', 'snippet'];
+
 const CHANNEL_ID = 'UCjycPAZuveusvPrk94-ClBw'; // This is Echo.Church's channel ID
-const API_KEY = Keys.YOUTUBE_RELEASE_API_KEY;
-const url =
-  'https://www.googleapis.com/youtube/v3/playlists?part=id%2CcontentDetails%2Csnippet';
+const API_KEY = Keys.YOUTUBE_DEV_API_KEY;
+const url = 'https://www.googleapis.com/youtube/v3';
+
+// this is good to keep around for logging axios req/resp
+/*
+axios.interceptors.request.use(request => {
+  console.log('Starting Request', request)
+  return request
+})
+
+axios.interceptors.response.use(response => {
+  console.log('Response:', response)
+  return response
+})
+*/
```

Comment:
you could guard this for dev env only 🤔 
```suggestion
if (__DEV__) {
  axios.interceptors.request.use(request => {
    console.log('Starting Request', request)
    return request
  })

  axios.interceptors.response.use(response => {
    console.log('Response:', response)
    return response
 })
}
```

Reply:
@dereckquock I don't want this to run in dev all the time--it fills up the output, but it's really convenient to have if you're debugging something related to fetching.